src/main/groovy/info/solidsoft/gradle/pitest/PitestPlugin.groovy

```diff
@@ -91,6 +92,13 @@ class PitestPlugin implements Plugin<Project> {
                 addPitDependencies(pitestConfiguration)
             }
         }
+        //for root project only
+        if (project.rootProject == project) {
```

Comment:
That will not work for monorepos. Alternatively, there could be a separate aggregate plugin (in the same JAR) applied on a desired (sub)project to check only the subprojects of that project. Nevertheless, that would probably complicate the implementation, so let's keep it as it and wait for the first request to improve it.

Reply:
I think separate plugin is good idea. Ive moving into this.